docs/pages/versions/unversioned/sdk/reanimated.md

```diff
@@ -15,7 +15,7 @@ import PlatformsSection from '~/components/plugins/PlatformsSection';
 
 <APIInstallSection href="https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation" />
 
-**In all cases,** after the installation completes, you must also add the Babel plugin to **babel.config.js**:
+**In all cases,** after the installation completes, you must also add the Babel plugin to either **.babelrc** and/or **babel.config.js**:
```

Comment:
```suggestion
**In all cases,** after the installation completes, you must also add the Babel plugin to your [Babel configuration file](https://babeljs.io/docs/en/config-files#configuration-file-types):
```

Let's link Babel docs instead of listing all possible variants of config file names/location.

Reply:
```suggestion
**In all cases,** after the installation completes, you must also add the Babel plugin to your [Babel configuration file](https://babeljs.io/docs/config-files#configuration-file-types):
```

Looks like Babel dropped i18n part of the path.